data_validation/config_manager.py

```diff
@@ -281,6 +281,20 @@ def threshold(self):
         """Return threshold from Config"""
         return self._config.get(consts.CONFIG_THRESHOLD, 0.0)
 
+    @property
+    def source_query(self):
```

Comment:
This should have a default value on None:
`return self._config.get("source_query", None)`

We may also want to have a consts here for SOURCE_QUERY and TARGET_QUERY so we can easily change the name in the future.

Reply:
Fixed in this [commit](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/78cbe136cd70a5208799a61a6c5602baff080e74)